src/components/Header.vue

```diff
@@ -0,0 +1,253 @@
+<template>
+  <header class="header-wrapper">
+    <nav class="header-wrapper-nav">
+      <h1 class="header-logo">Flux Academy</h1>
+      <ul class="header-wrapper-ul" :class="{ navActive: isOpen }">
+        <li>
+          <a
+            @click="activate(1)"
+            :class="{ active: active_el == 1 }"
+            href="#home-component-test"
+            >Inicio</a
+          >
+        </li>
+        <li>
+          <a
+            @click="activate(2)"
+            :class="{ active: active_el == 2 }"
+            href="#clases-component"
+            >Clases</a
+          >
+        </li>
+        <li>
+          <a
+            @click="activate(3)"
+            :class="{ active: active_el == 3 }"
+            href="#galeria-component"
```

Comment:
I used "gallery-component" as the div id for the gallery feature, are we going to use the name conventions in english or in spanish?

Reply:
In English, I just didn't want to leave it blank, I was waiting to see what you called it to change it.

Reply:
I have change it

Reply:
Great.